Dockerfile.dev

```diff
@@ -1,12 +1,12 @@
 #syntax=docker/dockerfile:1.2
-FROM strapi/base
+FROM node:20-alpine
 WORKDIR /srv/app
 COPY ./package.json ./
 COPY ./package-lock.json ./
-RUN npm run init
 COPY config ./config
 COPY public ./public
+COPY init ./init
 COPY src ./src
-RUN npm run build
+RUN npm run init || true
```

Comment:
Why don't we need the build step anymore?

Reply:
Init command takes care of it https://github.com/dev-launchers/strapiv4/pull/186/files#diff-405535515db7762623f57c6a4877ff8bac7d28b8e899146ad82e6ca522362f4fR18